tests/plugins/wysiwygarea/detached.js

```diff
@@ -0,0 +1,31 @@
+/* bender-tags: 4.17.0, bug, 4462 wysiwygarea */
+/* bender-ckeditor-plugins: basicstyles,toolbar */
+
+( function() {
+	bender.test( {
+		'Reattached editor contains same data': function() {
+			var startupData = '<p>CKEditor4</p>';
+
+			bender.editorBot.create( {
+				startupData: startupData
+			}, function( bot ) {
+				var editorContainer = bot.editor.container,
+					editorContainerParent = editorContainer.getParent();
+
+				editorContainer.remove();
+				editorContainerParent.append( editorContainer );
+
+				var iframeElement = bot.editor.ui.space( 'contents' ).findOne( 'iframe' );
+
+				CKEDITOR.tools.setTimeout( function() {
```

Comment:
I tried to find any, but I couldn't ;(  
In the end, it may end with hacking each browser event call. Similar problem I have during development, but fortunately was able to find one :)  
  
This time, it's less messy to just wait a little bit.